nemo/collections/common/metrics/perf_metrics.py

```diff
@@ -33,6 +44,19 @@ class FLOPsMeasurementCallback(Callback):
 
     higher_is_better = True
 
+    def __new__(cls, model_config, *args, **kwargs):
+        create_tensorboard_logger = model_config.get('exp_manager', {}).get("create_tensorboard_logger", False)
+        # Currently, supports TensorBoard logger only. Add check for new loggers when supported
+        if not create_tensorboard_logger:
+            logging.warning(
+                "TensorBoard logger is not enabled. FLOPsMeasurementCallback currently supports Tensorboard logger only"
+            )
+            logging.info("Enable TensorBoard logger using 'exp_manager.create_tensorboard_logger=True'")
+            return NoOpCallback()
```

Comment:
Should we throw an exception as it might a silent failure if user is not paying attention to the log ? @ericharper  for input 

(Apologies for second thoughts)